docs/Manage-Space.rst

```diff
@@ -293,28 +298,64 @@ One of the two following cases occurs when you select this tab.
 
 |image27|
 
+.. _SpaceHomePortlets:
+
+**Space home portlets**
+
+Three portlets are available for space homepage:
+
+-  Description: should display the space's description and the list of the space's managers.
+-  :ref:`Calendar <CalendarApp>`: displays the today's events that are added to the space's calendar.
+-  :ref:`Who is on line? <WhoIsOnlineApp>`: displays the on line members of the space.
+
+.. note:: :ref:`Calendar <CalendarApp>` and :ref:`Who is on line? <WhoIsOnlineApp>` portlets are the same
+          as for :ref:`intranet homepage <PLFHomepage>` but specific for the space's calendar and members.
+
+By default, only two portlets appear in the newly created space: **Description** and **Calendar**.
+
+-  Description portlet displays the space's description inputted during :ref:`the space creation <second_step>` 
+   and the space's creator full name with his avatar.
+   
+   |image76|
+
+-  Calandar portlet is empty by default as there are no events.
+    
+   |image77|
+
+If you :ref:`invited users during the space's creation <Users_invitation>` and that they accept the invitation, 
+:ref:`Who is on line? <WhoIsOnlineApp> portlet appears to display their avatars if they are not choosing the
+:ref:`invisible chat status <SettingStatus>`.
```

Comment:
No need to mention `invisible chat status` here IMO, it is confusing.

Reply:
OK

Reply:
OK